common/user.js

```diff
@@ -460,6 +469,71 @@ class User {
     });
   };
 
+  /**
+   * The Arkime user-info object (information provided to roleAssigners).
+   *
+   * @typedef ArkimeUserInfo
+   * @type {object}
+   * @param {string} userId - The ID of the user.
+   * @param {string} userName - The name of the user (to be displayed in the UI).
+   * @param {boolean | undefined} hasRole - whether the user has the requested role
+   *            (only if a role was provided & the requester is a roleAssigner for it)
+   */
+
+  /**
+   * POST - /api/users/list
+   *
+   * Retrieves a list of users (non-admin usable [with role status returned only for roleAssigners]).
+   * @name /users
+   * @returns {boolean} success - True if the request was successful, false otherwise
+   * @returns {ArkimeUserInfo[]} data - The list of users configured.
+   * @returns {number} recordsTotal - The total number of users.
+   * @returns {number} recordsFiltered - The number of users returned in this result.
+   */
+  static apiGetAssignableUsers (req, res, next) {
+    const query = {
+      from: +req.body.start || 0,
+      size: +req.body.length || 10000,
+      noRoles: true,
+      sortField: 'userId',
+      sortDescending: false
+    };
+    if (req.body.filter) { query.filter = req.body.filter; }
+    query.searchFields = ['userId', 'userName'];
+
+    Promise.all([
+      User.searchUsers(query),
+      User.numberOfUsers()
```

Comment:
i dont see the counts in the UI, do we need the numberOfUsers call and data?

Reply:
Oops, looks like I forgot to remove that from back when I was still planning on using pagination.